Emby.Naming/Video/ExtraRuleResolver.cs

```diff
@@ -69,7 +70,8 @@ public static ExtraResult GetExtraInfo(string path, NamingOptions namingOptions)
                 else if (rule.RuleType == ExtraRuleType.DirectoryName)
                 {
                     var directoryName = Path.GetFileName(Path.GetDirectoryName(pathSpan));
-                    if (directoryName.Equals(rule.Token, StringComparison.OrdinalIgnoreCase))
+                    string fullDirectory = Path.GetDirectoryName(pathSpan).ToString();
+                    if (directoryName.Equals(rule.Token, StringComparison.OrdinalIgnoreCase) && fullDirectory != libraryRoot)
```

Comment:
```suggestion
                    if (directoryName.Equals(rule.Token, StringComparison.OrdinalIgnoreCase)
                        && !string.Equals(fullDirectory, libraryRoot, StringComparison.OrdinalIgnoreCase))
```

Reply:
@crobibero Latest commit adds tests for this fix, as well as some tests for other audio/video extra folder names that weren't currently being checked.